guarddog/analyzer/analyzer.py

```diff
@@ -139,11 +140,87 @@ def analyze_sourcecode(self, path, rules=None) -> dict:
         Returns:
             dict[str]: map from each source code rule and their corresponding output
         """
+        semgrepscan_results = self.analyze_semgrep(path, rules)
+
+        yarascan_results = self.analyze_yara(path, rules)
+
+        # Concatenate dictionaries together
+        issues = semgrepscan_results["issues"] + yarascan_results["issues"]
```

Comment:
should this be a `|` instead of `+` like in the lines below?

Reply:
no, issues is a number, it represents the amount of issues identified in the package, that's it is adding both amounts of issues founds